### PR TITLE
remove incorrect urls pointing to the llava repository

### PR DESCRIPTION
**Pull request by BiliBraker**
# What does this PR do?
This PR removes 4 incorrect URLs pointing to the LLaVA repository.

Fixes # (issue)
https://github.com/huggingface/transformers/issues/31101

## Before submitting
- [ x ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.
@NielsRogge

### CODE REVIEW

File: src/transformers/models/idefics2/modeling_idefics2.py [hunk 1]
```diff
@@ -1345,8 +1345,7 @@ class Idefics2PreTrainedModel(PreTrainedModel):
 
     def _init_weights(self, module):
         # important: this ported version of Idefics2 isn't meant for training from scratch - only
-        # inference and fine-tuning - so the proper init weights code has been removed - the original codebase
-        # https://github.com/haotian-liu/LLaVA/tree/main/idefics2 should serve for that purpose
+        # inference and fine-tuning - so the proper init weights code has been removed
```
Comment (NielsRogge) [inline src/transformers/models/idefics2/modeling_idefics2.py:1348]: This comment can be entirely removed (same for other models)

**Review by amyeroberts — APPROVED** [review]
Thanks!

**Comment by amyeroberts** [comment]
@BiliBraker For the quality checks, could you run `make fix-copies` and push the changes?

**Comment by amyeroberts** [comment]
@BiliBraker Running `make fixup` should fix the failing quality checks

**Pull request merged**